packages/rn-tester/android/app/build.gradle

```diff
@@ -154,6 +154,11 @@ android {
     if (rootProject.hasProperty("ndkVersion")) {
         ndkVersion rootProject.ext.ndkVersion
     }
+    // For M1 Users we need to use the NDK 24, otherwise we default to the
+    // side-by-side NDK version from AGP.
+    if (System.properties['os.arch'] == "aarch64") {
+        ndkVersion = "24.0.8215888"
+    }
```

Comment:
this shoudl not be needed as we do have this config at the top level Gradle file already:

https://github.com/facebook/react-native/blob/0.69-stable/template/android/build.gradle#L12-L18

Reply:
My bad, I thought your change was in the template while the issue in on the template.